drivers/gnss/CMakeLists.txt

```diff
@@ -5,3 +5,5 @@ zephyr_library()
 zephyr_library_sources(gnss_publish.c)
 zephyr_library_sources_ifdef(CONFIG_GNSS_PARSE gnss_parse.c)
 zephyr_library_sources_ifdef(CONFIG_GNSS_NMEA0183 gnss_nmea0183.c)
+zephyr_library_sources_ifdef(CONFIG_GNSS_NMEA0183_MATCH gnss_nmea0183_match.c)
+zephyr_library_sources_ifdef(CONFIG_QUECTEL_LCX6G quectel_lcx6g.c)
```

Comment:
this should be in the next patch, but also, how about keeping the `GNSS_` prefix for all the drivers as well? Also consider the file names, I think most subsystem use a prefix for those as well.

Reply:
Could even use `gnss_lx6g.c`

Reply:
I updated the names to `GNSS_QUECTEL_LCX6G` and named the source file `gnss_quectel_lcx6g`. Some modems have very simplistic names like M8, so I want to keep the vendor prefix :)